pos_return_order/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+PoS - Return Order
+==================
+
+This module extends the functionality of odoo Point Of Sale about POS Order
+returns.
+
+With this module, it is now forbidden to return more quantity than the initial
+one.
+
+A link is created between the returned Order and the initial Order.
+A link is created between the returned Order Line and the initial Order Line.
+
+Implemented Features
+====================
+
+* A wizard that allow to select just some products to return:
+
+.. image:: /pos_return_order/static/description/partial_return_wizard.png
```

Comment:
I am wondering whether the link works in all circumstances (apps and inside Odoo at least), usually this is `./static/description/partial_return_wizard.png` or `static/description/partial_return_wizard.png`